pkg/reconcilermanager/controllers/reconciler_base_test.go

```diff
@@ -652,6 +653,7 @@ func TestMountConfigMapValuesFiles(t *testing.T) {
 								LocalObjectReference: corev1.LocalObjectReference{
 									Name: "foo",
 								},
+								Optional: &tr,
```

Comment:
pointer.Bool(true)

Reply:
🤦‍♀️ Ah my bad, missed that one, I'll update it

Reply:
done